setup.py

```diff
@@ -14,6 +14,7 @@
 install_requires = [
     'aiohttp>=3.6.2',
     'beautifulsoup4>=4.8.2',
+    'lxml'
```

Comment:
This is an extra dependency which is probably not strictly required to use the package - also the version is not pinned. It would probably be better to allow the user to provide the html.parser implementation they want and default to 'html.parser' if an implementation is not provided.